docs/installation.rst

```diff
@@ -190,19 +188,17 @@ This uses the latest (source package) release from PyPi.
                                # 0.41 and 0.41.1 have unicode issues at install time
     pip install borgbackup
 
-Note: we install into a virtual environment here, but this is not a requirement.
+.. note:: We install into a virtual environment here, but this is not a requirement.
 
 
 Installation (git)
 ------------------
 
 This uses latest, unreleased development code from git.
-While we try not to break master, there are no guarantees on anything.
-
-.. parsed-literal::
+While we try not to break master, there are no guarantees on anything. ::
 
-    # get |project_name| from github, install it
-    git clone |git_url|
+    # get borg from github
+    git clone https://github.com/borgbackup/borg.git
```

Comment:
Sphinx doesn't do any highlighting for the `parsed-literal` block and the CSS style for non-highlighted blocks is different

![docs](https://cloud.githubusercontent.com/assets/2482259/10461168/693e2caa-71d8-11e5-8c42-a218d088c5b4.jpg)